packages/documentation/src/stories/foundation/accessibility/accessibility.docs.mdx

```diff
@@ -12,6 +12,18 @@ import liveRegionBadAngularSample from './liveregion-bad-angular.sample.html?raw
   TODO: write accessibility page
 </div>
```

Comment:
@davidritter-dotcom can you remove this alert and all the empty sections that are TODOs? I'll create tickets to write these docs.